packages/orchestration/src/examples/swapExample.contract.js

```diff
@@ -41,7 +41,7 @@ export const start = async (zcf, privateArgs) => {
     // eslint-disable-next-line no-shadow -- this `zcf` is enclosed in a membrane
     async (/** @type {Orchestrator} */ orch, { zcf }, seat, offerArgs) => {
       const { give } = seat.getProposal();
-      !AmountMath.isEmpty(give.USDC.value) || Fail`Must provide USDC.`;
+      !AmountMath.isEmpty(give.USDC) || Fail`Must provide USDC.`;
```

Comment:
This check is redundant with the proposal shape below. It doesn't hurt correctness, of course, but I'm not sure it's exemplary.

Reply:
Good point. Maybe that was written before the invitation had a shape guard. I'll remove the line.